spec/models/export_spec.rb

```diff
@@ -38,6 +38,18 @@
       expect(results.first).to eq 'Account address,Show boosts,Notify on new posts,Languages'
       expect(results.second).to eq 'one@local.host,true,false,'
     end
+
+    it 'returns a csv of the following tags' do
```

Comment:
method under test should be described
```suggestion
    describe '#to_following_tags_csv' do
      it 'should return a csv of the following tags' do
```